initializr-service/application.yml

```diff
@@ -456,6 +462,25 @@ initializr:
           versionRange: 1.3.0.M4
           groupId: org.springframework.cloud
           artifactId: spring-cloud-cluster-hazelcast
+    - name: Spring Cloud Services for Pivotal Cloud Foundry
+      bom: scs-bom
+      versionRange: "1.0.1.RELEASE"
```

Comment:
the `versionRange` is always the Spring Boot version. I can polish the PR before merging.